packages/core/src/config/config.ts

```diff
@@ -125,20 +125,26 @@ export type ResolveConfig = {
 export const DEFAULT_REGION = 'us';
 export type Region = 'us' | 'eu';
 export type AccessTokens = {[region in Region]?: string };
-const REDOCLY_DOMAIN = process.env.REDOCLY_DOMAIN;
-export const DOMAINS: { [region in Region]: string } = {
-  us: 'redoc.ly',
-  eu: 'eu.redocly.com',
-};
 
-// FIXME: temporary fix for our lab environments
-if (REDOCLY_DOMAIN?.endsWith('.redocly.host')) {
-  DOMAINS[REDOCLY_DOMAIN.split('.')[0] as Region] = REDOCLY_DOMAIN;
-}
-if (REDOCLY_DOMAIN === 'redoc.online') {
-  DOMAINS[REDOCLY_DOMAIN as Region] = REDOCLY_DOMAIN;
+export const env = process?.env || {};
```

Comment:
`process` can be not defined as well. I think you need 

```suggestion
export const env = typeof process !== 'undefined' ? process?.env || {} : {};
```

Reply:
Good catch, fixed.